ppx/ppx_mysql.ml

```diff
@@ -1,268 +1,280 @@
 open Base
 open Ppxlib
-
 module Used_set = Caml.Set.Make (String)
 module Buildef = Ast_builder.Default
 
-
 (********************************************************************************)
 (** {1 Type definitions}                                                        *)
+
 (********************************************************************************)
 
 type param =
-    {
-    typ: string;
-    opt: bool;
-    name: string;
-    of_string: string * string;
-    to_string: string * string;
-    }
+  { typ : string
+  ; opt : bool
+  ; name : string
+  ; of_string : string * string
+  ; to_string : string * string }
```

Comment:
Weird, but i can live with this.

Reply:
I agree. Found no option to change it though. If this turns out to be the community consensus I guess I can live with it too.